### PR TITLE
Fix cicd install bundler

### PR DESCRIPTION
**Pull request by ztefanie**
### Short description
delivery is failing due to wrong permissions while installing the gems, see here: https://app.circleci.com/pipelines/github/digitalfabrik/entitlementcard/4572/workflows/75290bc2-3332-49b1-8762-74c3ea5ab9e9/jobs/35497

### Proposed changes

- fixed the duplication of messages sent
- added missing check step to beta delivery workflow
- fixed bundler update issue
- updated android images

### CODE REVIEW

File: .circleci/src/commands/notify.yml [hunk 1]
```diff
@@ -31,7 +31,10 @@ steps:
...
       command: |
         echo 'export ALLOW_ALL_BRANCHES="<< parameters.allow-all-branches >>"' >> $BASH_ENV
-        app-toolbelt v0 notify mattermost --message "${MM_MESSAGE}" --channel << parameters.channel >> ${ALLOW_ALL_BRANCHES:+--allow-all-branches}
-      name: Mattermost Status Alert
+      name: Set ALLOW_ALL_BRANCHES flag
       when: always
-
+  - run:
+      command: |
+        app-toolbelt v0 notify mattermost --message "<< parameters.success_message >>" --channel << parameters.channel >> <<#parameters.allow-all-branches>>--allow-all-branches<</parameters.allow-all-branches>>
```
Comment (f1sh1918) [inline .circleci/src/commands/notify.yml:38]: is this "#" correct here? `#parameters.allow-all-branches`

**Review by f1sh1918 — COMMENTED** [review]
Looks fine to :)
Just one comment

File: .circleci/src/commands/notify.yml (hunk 1 above)
Comment (ztefanie) [reply .circleci/src/commands/notify.yml:38]: yes it is correct, this is some circleci magic syntax to add the content between the both `<< >>` if the parameter is true.

Comment (f1sh1918) [reply .circleci/src/commands/notify.yml:38]: yes i know. but i'm talking about the additional `#` what does that mean?

**Review by f1sh1918 — APPROVED** [review]
👍 nice work :) thx for fixing

**Pull request merged**